modules/compliance-applying-resource-requests-and-limits.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+:_content-type: CONCEPT
+[id="compliance-applying-resource-requests-and-limits"]
+= Applying resource requests and limits
```

Comment:
This is all correct and useful information, but I wonder if instead of being part of the CO docs we could have all this info in some generic OCP doc? All that is described here applies equally well outside the CO world..

Reply:
That is definitely part of the plan. William and I are part of the larger Operator and OSDK docs team as well. The plan, for now, is to get this content available to CO users while we determine where it can also be visible in the Operator/OSDK docs too. Ultimately, it will be a one-line change to a file to insert this module into the Operator/OSDK docs when the time comes!